packages/@uppy/react/src/getHTMLProps.js

```diff
@@ -0,0 +1,171 @@
+// Attributes in the format of htmlAtrribute: reactAttribute
+const possibleStandardNames = {
```

Comment:
It looks like only the react-style names (the values in this object) are actually needed. Could we use an array instead?